common/rendering.h

```diff
@@ -600,6 +601,30 @@ namespace rs2
                         glTexImage2D(GL_TEXTURE_2D, 0, GL_RGB, width, height, 0, GL_LUMINANCE_ALPHA, GL_UNSIGNED_BYTE, data);
                     }
                     break;
+                case RS2_FORMAT_NV12I:
+                    if (nv122rgb)
```

Comment:
I think we can also change `nv122rgb` to nv12i_to_rgb, will be more readable right?

Reply:
Done